### PR TITLE
Make admin group configurable

### PR DESCRIPTION
**Pull request by awead**
Use `Ability#admin?` so you can override to provide your own logic.

@projecthydra/sufia-code-reviewers

### CODE REVIEW

File: app/search_builders/curation_concerns/search_filters.rb [hunk 1]
```diff
@@ -6,7 +6,7 @@ module CurationConcerns::SearchFilters
   # Override Hydra::AccessControlsEnforcement (or Hydra::PolicyAwareAccessControlsEnforcement)
   # Allows admin users to see everything (don't apply any gated_discovery_filters for those users)
   def gated_discovery_filters(permission_types = discovery_permissions, ability = current_ability)
-    return [] if ability.current_user.groups.include? 'admin'
+    return [] if ability.current_user.groups.include? CurationConcerns.config.admin_group
```
Comment (jcoyne) [inline app/search_builders/curation_concerns/search_filters.rb]: how about `ability.admin?`

Comment (awead) [inline app/search_builders/curation_concerns/search_filters.rb]: works for me.

**Comment by mjgiarlo** [comment]
Happy to :shipit: once the `SearchFilters` module change is made.

**Comment by awead** [comment]
@mjgiarlo @jcoyne this has been updated.

**Pull request merged**